src/tester.rs

```diff
@@ -483,8 +483,13 @@ fn run_tests(
                     if pass_stdout.is_none() || pass_stdout == Some(false) {
                         failure.stdout = Some(stdout_utf8);
                     }
+
+                    // If a test failed, bail out immediately, otherwise subsequent tests will
```

Comment:
So should we clarify that we are talking about sub-tests, not entire tests?

Reply:
Agreed.